docs/contributor-guide/developer-guide/technical-contribution-guide.md

```diff
@@ -147,25 +147,30 @@ If you're editing files that others have merged changes to you may need to resol
 
 When you're ready to have your code reviewed, open a pull request. After pushing a new branch there should be a "Compare & pull request" button on the github repository. Alternatively, go to the [new pull request](https://github.com/civiform/civiform/compare) page in github.
 
-Include "Closes #X" in the description to link the relevant issue.
-
-### Getting reviews on a pull request
+#### Guidelines for pull requests
+- Ensure the PR references the relevant issue it relates to in the PR comment. At the bottom of the PR template is a place to add "Fixes #X" which will link your PR to the issue and close it once the PR is merged.
+- Remove checkboxes from the PR template that don't apply. Ensure all checkboxes are checked before a PR is merged.
+- If a PR is inconsequential for deployments, like changes to github templates or actions, code changes without user-visible effects, etc., add the `ignore-for-release` label, so it is not included in the release notes. If they are deployment tool-specific or infrastructure-specific (read: dev tooling) changes, add `deploy` or `infrastructure` labels, as these will also get removed from the release notes.
+- If your PR includes a database evolution, include the `database` label.
+- A PR should be opened as Draft until it is ready for review.
+- A PR is ready for review when it is code complete, unit and browser tests are created (where appropriate), `bin/fmt` has been run, all checkboxes listed in the PR template that are relevant are checked and completed, all PR checks are passing, and manual testing has been performed.
+- Manual testing should ideally be performed with both Chrome and Firefox if your PR includes a front-end change.
+- If appropriate, add instructions in PR comments for reviewers on how to manually test it.
+- Press the "re-request review" button (the "arrows pointing in a circle" icon next to the reviewer's name in the Reviewers section on the Conversation tab) when you've fixed all the things mentioned in the comments and are ready for the reviewers to re-review.
```

Comment:
suggestion (non-blocking): Add language about posting in the the #pull-request-discuss channel and pinging the reviewer (optional?)